hack/local-up-cluster.sh

```diff
@@ -139,7 +139,7 @@ if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
 fi
 
 if [ "$(id -u)" != "0" ]; then
-    echo "WARNING : This script MAY be run as root for docker socket / iptables functionality; if failures occur, retry as root." 2>&1
+    echo "WARNING : This script MAY be run as root for docker socket functionality; if failures occur, retry as root." 2>&1
 fi
```

Comment:
We can remove this entire line, from what i have tested, the script is working without root

Reply:
+1, I have tested this as well and we can do without the warning

Reply:
sounds good @adisky